recipe/patches/0001-fix-test-cmake-config.patch

```diff
@@ -0,0 +1,187 @@
+From 873a321689f75834898020641aae460dfc78f1ed Mon Sep 17 00:00:00 2001
```

Comment:
This commit doesn't match https://patch-diff.githubusercontent.com/raw/scikit-build/scikit-build-core/pull/847.patch. Also, this patch is was released in v0.10.0 I think.